explorer/lib/explorer/aligned_layer_service_manager.ex

```diff
@@ -133,6 +134,7 @@ defmodule AlignedLayerServiceManager do
           response_transaction_hash: batch_response.transaction_hash,
           response_timestamp: batch_response.block_timestamp,
           amount_of_proofs: unverified_batch.amount_of_proofs,
+          proof_hashes: nil #don't need this value to update an existing but unverified batch, it is on another table
```

Comment:
What does it mean?
> don't need this value to update an existing but unverified batch, it is on another table

Reply:
it means it's ok for that function to return the struct with `nil` in its proof_hashes parameter